docs/se/guides/tracing.adoc

```diff
@@ -115,68 +114,86 @@ section (*not* `<dependencyManagement>`). This will enable Helidon to use Jaeger
 ----
 tracing:
   service: helidon-se-1
+  protocol: http
+  port: 14250
+  path: /api/traces
+  tags:
+    env: development
+  enabled: true
+  sampler-type: "const"
+  sampler-param: 1
+  log-spans: true
+  propagation: b3
 ----
 
 [source,java]
-.Update the `Main` class; Add Tracer to the WebServer builder
+.Update the `Main` class. Add Tracer to the WebServer builder
 ----
-import io.helidon.tracing.TracerBuilder; // <1>
+import io.helidon.tracing.TracerBuilder;
 ...
+
+Tracer tracer = TracerBuilder.create("helidon") <1>
+                .build();
+
 WebServer server = WebServer.builder(createRouting(config))
                 .config(config.get("server"))
-                .tracer(TracerBuilder.create(config.get("tracing")).build()) // <2>
+                .addFeature(ObserveFeature.builder()
+                            .addObserver(TracingObserver.create(tracer)) <2>
+                            .build())
                 .addMediaSupport(JsonpSupport.create())
                 .build();
 ----
-<1> Add a new import statement.
-<2> Build and register a `Tracer` object using the tracing configuration.
+<1> Create the `Tracer` object.
+<2> Add an observability feature using the created `Tracer`.
 
 [source,java]
 .Update the `GreetService` class; 1) Add a new import and 2) Replace the `getDefaultMessageHandler` method:
 ----
-import io.opentracing.Span; // <1>
-...
-    private void getDefaultMessageHandler(ServerRequest request,
-                                   ServerResponse response) {
+private void getDefaultMessageHandler(ServerRequest request,
+                                      ServerResponse response) {
+    var spanBuilder = Tracer.global().spanBuilder("mychildSpan"); <1>
+    request.context().get(SpanContext.class).ifPresent(sc -> sc.asParent(spanBuilder)); <2>
+    var span = spanBuilder.start(); <3>
 
-        var spanBuilder = request.tracer()  // <2>
-                .buildSpan("getDefaultMessageHandler");  // <3>
-        request.spanContext().ifPresent(spanBuilder::asChildOf);  // <4>
-        Span span = spanBuilder.start();  // <5>
-
-        try {
-            sendResponse(response, "World");
-        } finally {
-            span.finish();  // <6>
-        }
+    try {
+        sendResponse(response, "World");
+        span.end(); <4>
+    } catch (Throwable t) {
+        span.end(t);    <5>
     }
+}
 ----
-<1> Add new import statement.
-<2> Get the `Tracer` object from the request.
-<3> Build a new span named `getDefaultMessageHandler`.
-<4> Make the new span a child of the current span.
-<5> Start the span.  The current timestamp is used as the starting time for the span.
-<6> Finish the span.  The current timestamp is used as the ending time for the span.
-
+<1> Create a new `Span` using Global Tracer.
+<2> Set Parent Span, if available in the `Request`.
+<3> Start the span.
+<4> End the span when the response is sent.
+<5> Record an exception in the span if happened.
```

Comment:
Very small changes mostly in upper/lower case:

<1> Create a new `Span` using the global tracer.
<2> Set the parent of the new span to the span from the `Request` if available.
<3> Start the span.
<4> End the span normally after the response is sent.
<5> End the span with an exception if one was thrown.

Reply:
ok :)